README.md

```diff
@@ -18,7 +18,7 @@ npm install --save hoodie-server-store
 ## Example
 
 ```js
-var Hapi = require('hapi')
+var hapi = require('hapi')
```

Comment:
why the lower case? In all examples I have seen, the `Hapi` varibale is always with a capital `H`, e.g. http://hapijs.com/tutorials#creating-a-server